packages/fast-components-react-msft/src/text-action/text-action.props.ts

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import { ManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
+import { TextActionClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+import {
+    TextFieldHandledProps,
+    TextFieldManagedClasses,
+} from "@microsoft/fast-components-react-base";
+import { Omit, Subtract } from "utility-types";
+import { ButtonAppearance } from "../button/button.props";
+
+export enum ButtonPosition {
```

Comment:
Well, it's the optional button that can be added to the text-action. `TextActionButtonPosition` seems unnecessarily verbose, but that's the only other name that comes to mind...

Reply:
It might seem unnecessarily verbose but remember these are global names. What is to differentiate this enum from an enum exported to be used directly by the button component?

Reply:
Totally - good point. Will change.